Sources/XcodeExport/XcodeTypographyExporter.swift

```diff
@@ -80,30 +90,25 @@ final public class XcodeTypographyExporter: XcodeExporterBase {
         ])
         return try makeFileContents(for: contents, url: swiftUIFontExtensionURL)
     }
-    
-    private func makeLabelStyleExtensionFileContents(textStyles: [TextStyle], labelStyleExtensionURL: URL) throws -> FileContents {
-        let dict = textStyles.map { style -> [String: Any] in
-            let type: String = style.fontStyle?.textStyleName ?? ""
-            return [
-                "className": style.name.first!.uppercased() + style.name.dropFirst(),
-                "varName": style.name,
-                "size": style.fontSize,
-                "supportsDynamicType": style.fontStyle != nil,
-                "type": type,
-                "tracking": style.letterSpacing.floatingPointFixed,
-                "lineHeight": style.lineHeight ?? 0,
-                "textCase": style.textCase.rawValue
-            ]}
-        let env = makeEnvironment(templatesPath: output.templatesPath)
-        let contents = try env.renderTemplate(name: "LabelStyle+extension.swift.stencil", context: ["styles": dict])
-        
-        let labelStylesSwiftExtension = try makeFileContents(for: contents, url: labelStyleExtensionURL)
-        return labelStylesSwiftExtension
-    }
-    
-    private func makeLabel(textStyles: [TextStyle], labelsDirectory: URL, separateStyles: Bool) throws -> FileContents {
```

Comment:
Looks like we are generating files for `style` and `style+extension ` files based on the fontSystem, but if it is UIKit, we are not generating `Label.swift` anymore?